src/pages/index.tsx

```diff
@@ -0,0 +1,611 @@
+            <ImageContainer>
+              <Image
+                src={future}
+                alt={t("page-index:page-index-internet-image-alt")}
+                width={700}
+              />
+            </ImageContainer>
+          </Row>
+        </Box>
+      </MainSectionContainer>
+      {/* Developer Section */}
+      <MainSectionContainer containerBg="homeBoxPurple">
+        <Row>
+          <Box py={4} px={{ base: 4, sm: 8 }} width="full">
+            <StyledTitleCardList
+              content={codeExamples}
+              clickHandler={toggleCodeExample}
+              headerKey="page-index:page-index-developers-code-examples"
+              isCode
+              border="1px"
+              borderColor="text"
+              boxShadow={cardBoxShadow}
+              maxWidth={{ lg: "624px" }}
+              ml={{ lg: 16 }}
+            />
+          </Box>
+          <FeatureContent>
+            <SectionHeading>
+              <Translation id="page-index:page-index-developers" />
+            </SectionHeading>
+            <SectionDecription>
+              <Translation id="page-index:page-index-developers-description" />
+            </SectionDecription>
+            <ButtonLinkRow>
+              <ButtonLink to="/dapps/?category=technology">
+                <Translation id="page-index:page-index-developers-button" />
+              </ButtonLink>
+            </ButtonLinkRow>
+          </FeatureContent>
+          <StyledCodeModal
+            isOpen={isModalOpen}
+            setIsOpen={setModalOpen}
+            title={codeExamples[activeCode].title}
+            sx={{
+              ".modal-component-container": {
+                padding: 0,
+                left: 0,
+                right: 0,
+                bottom: 0,
+                top: "50%",
+              },
+              ".modal-component": {
+                maxWidth: "100%",
+                maxHeight: "50%",
+                padding: 0,
+              },
+              ".modal-component-content": {
+                marginTop: "3rem",
+                width: "100%",
+                overflow: "auto",
+              },
+            }}
+          >
+            <Codeblock
+              codeLanguage={codeExamples[activeCode].codeLanguage}
+              allowCollapse={false}
+              fromHomepage
+            >
+              {codeExamples[activeCode].code}
+            </Codeblock>
+          </StyledCodeModal>
+        </Row>
+      </MainSectionContainer>
+      {/* Eth Today Section */}
+      <GrayContainer>
+        <ContentBox>
+          <SectionHeading mt={12} mb={8} fontFamily="heading">
+            <Translation id="page-index:page-index-network-stats-title" />
+          </SectionHeading>
+          <SectionDecription>
+            <Translation id="page-index:page-index-network-stats-subtitle" />
+          </SectionDecription>
+        </ContentBox>
+        {/* // TODO: migrate stats fetching the necessary data on build time */}
```

Comment:
```suggestion
```

Reply:
I think this should stay in. @pettinarip mentioned this needs migration still in dev sync today